File: osu!KeysoundSplitter/Audio/BaseAudioTrackInfo.cs

namespace osuKeysoundSplitter.Audio
{
    public abstract class BaseAudioTrackInfo : AudioTrackInfo
    {
        public override int getBytesPerSample()
        {
            return (getBitsPerSample() / 8) * getChannels();
        }

        public override int getBytesPerSecond()
        {
            return getSampleRate() * getBytesPerSample();
        }

        public override string toString()
        {
            return "SampleRate=" + getSampleRate() + " Channels=" + getChannels()
                    + " BPS=" + getBitsPerSample();
        }
    }
}

File: src/com/damnae/osukeysoundsplitter/audio/BaseAudioTrackInfo.java
package com.damnae.osukeysoundsplitter.audio;

public abstract class BaseAudioTrackInfo implements AudioTrackInfo {

	@Override
	public int getBytesPerSample() {
		return (getBitsPerSample() / 8) * getChannels();
	}

	@Override
	public int getBytesPerSecond() {
		return getSampleRate() * getBytesPerSample();
	}

	@Override
	public String toString() {
		return "SampleRate=" + getSampleRate() + " Channels=" + getChannels()
				+ " BPS=" + getBitsPerSample();
	}
}
